params.py

```diff
@@ -130,6 +130,7 @@
         'facenet': (Architecture.FACENET, None),
         'fbdeepface': (Architecture.FBDEEPFACE, None),
         'vggface': (Architecture.VGGFACE, None),
+        'keras_vggface': (Architecture.KERAS_VGGFACE, None),
         'arcface': (Architecture.ARCFACE, None),
```

Comment:
keras_vggface_resnet er ook bij?

Reply:
Keras ResNet geeft bij mij nog hele rare resultaten (cllr van 1, auc van 0.5, etc), bij jou ook? Misschien nog beter om die even helemaal weg te laten?

Reply:
ja hij kan er bij mij ook niks van :P maar ik dacht als we hem wel geimplementeerd hebben, kunnen we hem ook wel hierbij zetten